roles/validate/files/defaults.yml

```diff
@@ -35,6 +35,8 @@ factory_defaults:
         advertise_pip: false
         advertise_pip_border_only: true
         domain_id_range: 1-1000
+        fabric_vpc_qos: false
```

Comment:
Please open a pr to add these to the internal repo defaults since that is the source of truth for defaults.  Reminds me that I need to make sure we refreshing the set of defaults from that repo for each release.

Reply:
I discovered that we indeed where a bit out of sync.  Opened a PR to address this

https://wwwin-github.cisco.com/netascode/nac-vxlan/pull/123